book/sample.md

```diff
@@ -1,18 +0,0 @@
-% Write Yourself a Roguelike
```

Comment:
Just to clarify and make the thinking explicit - there's no need for these sample files because the whole book is available now, correct?

Reply:
Yep, that's my thinking. I'm unsure of the original utility of the sample files, but I assume they were to offer potential customers a glimpse of the book. Since now people can read the entire book for free, I say trash the samples.